CitizenLabDotCo/citizenlab
# CL-1024: Content builder copy and UX tweaks

## Checklist

- [x] Added entry to change log

## How to test
- Navigate to the content builder
- Add an about box
- Check to see that you can see all the appropriate CTA buttons before it

## Links

- [CL-1024](https://citizenlab.atlassian.net/browse/CL-1024)

## How urgent is a code review?

End of day
